### PR TITLE
DOCS-9495 - azure sidecar guides

### PR DESCRIPTION
**Pull request by cswatt**
### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/serverless/azure_app_services/azure_functions/_index.md
Comment (brandondatadog) [inline content/en/serverless/azure_app_services/azure_functions/_index.md:1]: Can we move this into its own category?

Comment (brandondatadog) [reply content/en/serverless/azure_app_services/azure_functions/_index.md:1]: I.e. outside of app services and into its own 'Azure Functions' category on the sidebar

**Comment by duncanpharvey** [comment]
Can we move the Azure Functions docs to a separate PR and hold merging them for now? There are several major changes I'm planning in the upcoming weeks:

* Deprecation of datadog/sma packages and creation of new packages with clearer names that start the mini agent (instead of starting the mini agent in the tracer)
* Addition of automatic instrumentation of Azure Functions package in Python tracer. Which means we will deprecate the dd-azure-worker-extension package

Once these changes are ready I feel better about making the Azure Function docs public since customers won't be installing packages that we plan to deprecate shortly.

**Comment by cswatt** [comment]
moved Azure Functions content to another PR: https://github.com/DataDog/documentation/pull/26336

**Comment by brandondatadog** [comment]
image
Seems like there's a bit of a gap between these sections? Also is it possible to put the language selector under 'Application' instead of the smaller subsections?

**Review by brandondatadog — APPROVED** [review]
LGTM!

File: content/en/serverless/guide/azure_app_service_linux_containers_serverless_init.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+title: Instrument Azure App Service with serverless-init - Linux Containers
```
Comment (janine-c) [inline content/en/serverless/guide/azure_app_service_linux_containers_serverless_init.md:2]: This page isn't linked to from the Guides page - should it be?

File: content/en/serverless/guide/azure_app_service_linux_containers_serverless_init.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+{{% svl-init-php %}}
+
+{{< /programming-lang >}}
+{{< /programming-lang-wrapper >}}
+
+### 2. Configure your application
+
+Once the container is built and pushed to your registry, the last step is to set the required environment variables for the Datadog Agent:
+- `DD_API_KEY`: Datadog API key, used to send data to your Datadog account. It should be configured as a [Azure Secret][7] for privacy and safety issue.
```
Comment (janine-c) [inline content/en/serverless/guide/azure_app_service_linux_containers_serverless_init.md:85]:
```suggestion
- `DD_API_KEY`: Datadog API key, used to send data to your Datadog account. It should be configured as an [Azure Secret][7] for privacy and safety.
```

File: content/en/serverless/guide/azure_app_service_linux_containers_serverless_init.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+{{% aas-workflow-linux %}}
+
+## Additional configurations
+
+- **Advanced Tracing:** The Datadog Agent already provides some basic tracing for popular frameworks. Follow the [advanced tracing guide][2] for more information.
+
+- **Logs:** If you use the [Azure integration][1], your logs are already being collected. Alternatively, you can set the `DD_LOGS_ENABLED` environment variable to `true` to capture application logs through the serverless instrumentation directly.
+
+- **Custom Metrics:** You can submit custom metrics using a [DogStatsd client][3]. For monitoring Cloud Run and other serverless applications, use [distribution][8] metrics. Distributions provide `avg`, `sum`, `max`, `min`, and `count` aggregations by default. On the Metric Summary page, you can enable percentile aggregations (p50, p75, p90, p95, p99) and also manage tags. To monitor a distribution for a gauge metric type, use `avg` for both the [time and space aggregations][9]. To monitor a distribution for a count metric type, use `sum` for both the time and space aggregations.
```
Comment (janine-c) [inline content/en/serverless/guide/azure_app_service_linux_containers_serverless_init.md:105]:
```suggestion
- **Custom Metrics:** You can submit custom metrics using a [DogStatsD client][3]. For monitoring Cloud Run and other serverless applications, use [distribution][8] metrics. Distributions provide `avg`, `sum`, `max`, `min`, and `count` aggregations by default. On the Metric Summary page, you can enable percentile aggregations (p50, p75, p90, p95, p99) and also manage tags. To monitor a distribution for a gauge metric type, use `avg` for both the [time and space aggregations][9]. To monitor a distribution for a count metric type, use `sum` for both the time and space aggregations.
```

File: content/en/serverless/guide/azure_app_service_linux_containers_serverless_init.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+### Environment Variables
+
+| Variable | Description |
+| -------- | ----------- |
+|`DD_API_KEY`| [Datadog API Key][6] - **Required**|
+| `DD_SITE` | [Datadog site][4] - **Required** |
+| `DD_LOGS_ENABLED` | When true, send logs (stdout and stderr) to Datadog. Defaults to false. |
+| `DD_LOGS_INJECTION`| When true, enrich all logs with trace data for supported loggers in [Java][10], [Node.js][11], [.NET][12], and [PHP][13]. See additional docs for [Python][14], [Go][15], and [Ruby][16]. |
+| `DD_TRACE_SAMPLE_RATE`|  Controls the trace ingestion sample rate `0.0` and `1.0`|
```
Comment (janine-c) [inline content/en/serverless/guide/azure_app_service_linux_containers_serverless_init.md:119]: Not sure about this, but it reads a little oddly to me as-is?

```suggestion
| `DD_TRACE_SAMPLE_RATE`|  Controls the trace ingestion sample rate between `0.0` and `1.0`.|
```

File: content/en/serverless/guide/azure_app_service_linux_containers_serverless_init.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+| `DD_SOURCE`       | See [Unified Service Tagging][5].                                       |
+| `DD_TAGS`         | See [Unified Service Tagging][5].                                       |
+
+## Troubleshooting
+
+If you are not receiving traces or custom metric data as expected, enable **App Service logs** to receive debugging logs.
+
+{{< img src="serverless/azure_app_service/app-service-logs.png" style="width:100%;" >}}
+
+Share the content of the **Log stream** with [Datadog Support][14].
```
Comment (janine-c) [inline content/en/serverless/guide/azure_app_service_linux_containers_serverless_init.md:132]: Not sure if this is the right link?

**Review by janine-c — COMMENTED** [review]
Hey Cecilia, just suggested a link fix and some small readability suggestions. Let me know when it's ready for a final rubber stamp :)

**Review by janine-c — APPROVED** [review]
:shipit:❗

**Review by davidejones — APPROVED** [review]

**Pull request merged**